API_SPECS_DEVICE_PROPERTIES.md

```diff
@@ -0,0 +1,42 @@
+API specifications - Device Properties
+======================================
+
+## Get a list of device properties
+
+The list of device properties contains properties from all device models.
+
+Request:
+```
+GET /v1/deviceproperties
```

Comment:
please change the case to "deviceProperties"

Reply:
Shall we do the same for `devicemodels` in `devicemodels API` ?

Reply:
@maple10001 yes